mention requirement for outbound traffic for kic+konnect (#5781)
Kong/docs.konghq.com: app/konnect/runtime-manager/kic.md

* `kubectl` or `oc` (if you're working with OpenShift) installed and configured to communicate with your Kubernetes TLS
* [Helm 3](https://helm.sh/docs/intro/install/) installed

> 
> Also, as {{site.kic_product_name}} calls the {{site.konnect_short_name}}'s APIs, outbound traffic from the
> {{site.kic_product_name}}'s pods must be allowed to reach {{site.konnect_short_name}}'s `*.konghq.com` hosts.

Reviewer: ```suggestion
* Because {{site.kic_product_name}} calls {{site.konnect_short_name}}'s APIs, outbound traffic from {{site.kic_product_name}}'s pods must be allowed to reach {{site.konnect_short_name}}'s `*.konghq.com` hosts.
```

Reviewer: Should we also add this information to https://deploy-preview-5781--kongdocs.netlify.app/konnect/network/ under "Hostnames"?

Author: Yes, I think that's a good point. Updated the `/konnect/network` accordingly and linked to it from the 2 articles edited.
